index.css

```diff
@@ -0,0 +1,3 @@
+header {
+  color: blue;
+}
```

Comment:
- [ ] According to the project requirements, you are required to use a class selector.

Reply:
Thank you. I have made the corrections

Reply:
Can you check it out again or I will be required to submit it again

Reply:
https://github.com/dialite/hello-world/pull/1